fs/vfs/fs_poll.c

```diff
@@ -276,6 +277,62 @@ static inline int poll_teardown(FAR struct pollfd *fds, nfds_t nfds,
  * Public Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: poll_notify_common
+ *
+ * Description:
+ *   Notify the poll, this function should be called by drivers to notify
+ *   the caller the poll is ready.
+ *
+ * Input Parameters:
+ *   afds     - The fds array
+ *   nfds     - Number of fds array
+ *   eventset - List of events to check for activity
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+void poll_notify_common(FAR struct pollfd **afds, int nfds,
+                        pollevent_t eventset, uint32_t option)
+{
+  int i;
+  int semcount;
+  FAR struct pollfd *fds;
+
+  DEBUGASSERT(nfds >= 1);
+
+  for (i = 0; i < nfds; i++)
+    {
+      fds = afds[i];
+      if (fds == NULL)
+        {
+          continue;
+        }
```

Comment:
minor / optional:
I would better use `if (fds != NULL)` and avoid `continue`.

Reply:
Done